econet_hvac_odu.yaml

```diff
@@ -34,6 +34,26 @@ sensor:
     request_mod: none
     accuracy_decimals: 1
     entity_category: "diagnostic"
+  - platform: econet
+    name: "ODU Pressure Liquid"
+    id: odu_pressure_liquid
+    sensor_datapoint: PRES_LIQ
+    request_mod: none
```

Comment:
`request_mod` basically groups requests, which is necessary in some cases because response payloads can get too large. I'm not sure whether any change is needed here given that these are just numerical values rather than potentially long strings. @tronikos is the true expert on that.

Reply:
Both of these sensors have `request_mod: none` which is actually preferred if the thermostat already requests these since we get them for free without any additional load to the microcontroller. If you do need to request them yourself any number will work, even 0 as long as the response payload of all requested endpoints doesn't exceed some unknown limit. If I remember right, you simply not get any response when that happens. My recommendation is to avoid 0 and reserve it for the absolutely necessary endpoints to avoid breaking them. At the same time try limiting the total number of request mods to minimize the microcontroller load.